expo/expo
# [tools] Fix expo-bot changelog suggestion display

# Why

expo-bot displays the first changelog suggestion inline:

> <img width="719" alt="Screenshot 2022-04-29 at 09 03 05" src="https://user-images.githubusercontent.com/278340/165898736-3bc8aebd-ba55-4857-9314-b4a2274ab52d.png">


# How

- Include the bullet list in the newline (was inline before)
- As suggested, removed the note about changelog update being optional
- Removed unnecessary line-breaks in the message content

# Test Plan

- Make dummy changes in this PR
- Ensure the bot now displays them correctly (`et review`)
  > See `expo-bot` comment and my comment for comparison
- **Revert these changes before merge**
